synfig-studio/po/POTFILES.in

```diff
@@ -392,6 +392,8 @@ src/synfigapp/actions/layerfit.cpp
 src/synfigapp/actions/layerfit.h
 src/synfigapp/actions/layerlower.cpp
 src/synfigapp/actions/layerlower.h
+src/synfigapp/actions/layerlowermax.cpp
+src/synfigapp/actions/layerlowermax.h
```

Comment:
maybe name file and action to "layer to bottom"?

Reply:
I don't know I feel like the preposition "to" in the action name looks a bit odd. However, If you think it's important though I don't have a hard opinion here we can change it.

Reply:
@ice0  ?